src/py/_pyodide/_core_docs.py

```diff
@@ -108,6 +108,36 @@ def assign_to(self, to: Any):
         an ArrayBuffer view.
         """
 
+    def tomemoryview(self) -> memoryview:
+        """Convert the buffer to a memoryview.
+
+        Copies the data once. This currently has the same effect as :any:`to_py`.
```

Comment:
Could you add a note that this and below methods are only defined for buffers? As right not it's not explicit.